ruby/Ice/config/client.rb

```diff
@@ -9,14 +9,15 @@
 require_relative 'Greeter.rb'
 
 # Create an Ice communicator to initialize the Ice runtime. The communicator is destroyed at the end of the block
-# returned by Ice::initialize.
+# returned by Ice::initialize. This communicator is initialized using command line input and the "config.client"
+# file as its arguments.
```

Comment:
For the comment see:

 https://github.com/zeroc-ice/ice-demos/blob/39c3ecb81db7daa62b54c3298da542a3396bf7f1/csharp/Ice/Config/Client/Program.cs#L8-L11